docs/quickstart-cli.md

```diff
@@ -117,7 +117,8 @@ cat <<EOF > ~/.ratify/config.json
         "version": "1.0.0", 
         "plugins": [ 
             { 
-                "name": "oras"
+                "name": "oras",
+                "useHttp": true,
```

Comment:
why is this necessary now?

Reply:
Is this doc intended for local testing? `useHttp` is unnecessary for remote registry, but seems it's required to use the local registry as the doc specified.

Reply:
Ahh yes I see